target/linux/qualcommax/ipq807x/base-files/etc/board.d/02_network

```diff
@@ -77,12 +78,11 @@ ipq807x_setup_macs()
 	local label_mac=""
 
 	case "$board" in
-		linksys,mx4200v2)
+		linksys,mx4200v2|\
+		linksys,mx4300)
 			label_mac=$(mtd_get_mac_ascii devinfo hw_mac_addr)
-			for i in $(seq 3 5); do
-				[ "$(mtd_get_mac_ascii u_env eth${i}addr)" != "$label_mac" ] && lan_mac=$label_mac
-			done
-			[ "$(mtd_get_mac_ascii u_env eth2addr)" != "$label_mac" ] && wan_mac=$label_mac
+			[ "$(mtd_get_mac_ascii u_env eth1addr)" != "$label_mac" ] && wan_mac=$label_mac
```

Comment:
Why's it only testing eth1 and skip the {3,4,5} from the previous version? Aren't LAN port IDs still 3,4,5 from the dts?

Reply:
These devices use the same address for all interfaces. There is no point in testing one by one. Some MX series devices have different MAC addresses for eth1 and eth2-5 but they are later overwritten to the same ones. Sometimes these MAC addresses are not used at all, they are overwritten by data from the `devinfo` partition.